### PR TITLE
Add ideas for poliastro

### PR DESCRIPTION
**Pull request by Juanlu001**

### CODE REVIEW

**Comment by dpshelio** [comment]
And [this is how it looks](https://27-30926520-gh.circle-artifacts.com/0/site/gsoc/gsoc2019/index.html#/projects).

File: _projects/2019/poliastro/poliastro-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+poliastro is already a useful tool for people with Python knowledge who know how to use a third party
+library. However, to reach a wider range of users we should be able to create web applications
+that use poliastro in the backend, and make this process as easy and cheap as possible.
+
+There have been several ideas to create web applications using poliastro, but one potential
+limitation has been the installation itself. poliastro dependencies are complex and somewhat
+heavy, and there is also dependence on external data for things like planetary ephemerides.
+This sole thing prevents the deployment of poliastro in services like Amazon Web Services Lambda,
+as evidenced in [this issue](https://github.com/poliastro/poliastro/issues/444).
```
Comment (dpshelio) [inline _projects/2019/poliastro/poliastro-as-a-service.md]: It's not good practice to mention links like `this`/`here`/..., I would say:
```suggestion
as evidenced in [the issue #444](https://github.com/poliastro/poliastro/issues/444).
```

File: _projects/2019/poliastro/validation-framework.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ - SOCIS
+tags:
+ - python
+collaborating_projects:
+ - poliastro
+---
+
+#### Description
+
+_Validation is hard._ The current approach in poliastro is to add unit tests that check whether poliastro gives the same result as the solved examples of popular Astrodynamics textbooks (see and [http://docs.poliastro.space/en/latest/references.html#books-and-papers](http://docs.poliastro.space/en/latest/references.html#books-and-papers)).
```
Comment (dpshelio) [inline _projects/2019/poliastro/validation-framework.md]: see and <link>? I would change it for:
```suggestion
_Validation is hard._ The current approach in poliastro is to add unit tests that check whether poliastro gives the same result as the solved examples of [popular Astrodynamics textbooks](http://docs.poliastro.space/en/latest/references.html#books-and-papers).
```

File: _projects/2019/poliastro/validation-framework.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+However, poliastro is now a mature project and validating against textbook examples is not enough anymore,
+for several reasons:
+
+* It is being used more and more in companies and professional settings, which cannot afford failure
+* It has grown in feature set and can do more complicated things, which are out of the scope of undergraduate textbooks
+
+One example of this is the tests covering perturbations and low-thrust guidance laws
+(see https://blog.poliastro.space/2018/08/05/2018-08-05-google-summer-of-code-2018/
```
Comment (dpshelio) [inline _projects/2019/poliastro/validation-framework.md]:
```suggestion
(see [last year results](https://blog.poliastro.space/2018/08/05/2018-08-05-google-summer-of-code-2018/)
```
or similar to remove full link on the text.

File: _projects/2019/poliastro/validation-framework.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+##### 1st evaluation
+
+* Have settled on one of the tools for reference frame validation
+* Have a basic understanding of reference frames
+* Have converted the `Body` rotational elements to proper reference frames
+* Have all code, tests and documentation in GitHub
+
+##### 2nd evaluation
+
+* Have produced a table similar to http://www.astropy.org/coordinates-benchmark/summary.html for our planetary reference frames
```
Comment (dpshelio) [inline _projects/2019/poliastro/validation-framework.md]:
```suggestion
* Have produced a table similar to [astropy's](http://www.astropy.org/coordinates-benchmark/summary.html) for our planetary reference frames
```

File: _projects/2019/poliastro/validation-framework.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+
+* Have used a second method to validate the reference frames
+* Have started a conversation in Astropy to integrate our code upstream
+* Have all code, tests and documentation in GitHub
+
+#### Secondary goals
+
+* Fix any bugs that might appear regarding visualization, hyperbolic orbits and so forth
+* Review pull requests from other poliastro contributors, including the mentor
+* Maintain poliastro benchmarks https://blog.poliastro.space/poliastro-benchmarks/
```
Comment (dpshelio) [inline _projects/2019/poliastro/validation-framework.md]:
```suggestion
* Maintain [poliastro benchmarks](https://blog.poliastro.space/poliastro-benchmarks/)
```

File: _projects/2019/poliastro/validation-framework.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+* Give a talk about the work at some Python event
```
Comment (dpshelio) [inline _projects/2019/poliastro/validation-framework.md]: :+1:

**Comment by Juanlu001** [comment]
Applied all suggestions, thanks a lot!

File: _projects/2019/poliastro/earth-specific-capabilities.md [hunk 7]
```diff
@@ -0,0 +1,72 @@
+---
+name: earth-specific-capabilities
```
Comment (Juanlu001) [inline _projects/2019/poliastro/earth-specific-capabilities.md]:
```suggestion
name: Earth-specific capabilities
```

File: _projects/2019/poliastro/export-czml.md [hunk 8]
```diff
@@ -0,0 +1,71 @@
...
+requirements:
+ - Familiar with the JSON format
+ - Basic experience with JavaScript or TypeScript
+difficulty: medium
+issues:
+ - https://github.com/poliastro/poliastro/issues/117
+ - https://github.com/poliastro/poliastro/issues/287
+mentors:
+ - juanlu001
+ - newlawrence
```
Comment (Juanlu001) [inline _projects/2019/poliastro/export-czml.md]:
```suggestion
```

**Comment by Juanlu001** [comment]
Do I have to use the template and change the naming convention or is this PR okay?

**Comment by bsipocz** [comment]
The file naming is perfectly fine, but just as in previous years we will need at least two active mentors per projects for any of these ideas to make the selection.

**Comment by bsipocz** [comment]
I'm merging this in now, any adjustments can be added in follow-up PRs when needed.

Thanks @Juanlu001!

**Comment by dpshelio** [comment]
Funny how I click approved and you merged at the same time!! :)

**Comment by bsipocz** [comment]
Oh, I didn't see that, merely just wanted to clear out as many PRs from here as possible. (While it's super annoying I cannot review the webpage...)

**Comment by bsipocz** [comment]
Yes, I've checking up on that. Still it worries me a bit as students may run into the same issue that the page is not available. I thing we had a similar issue last year, too?

**Pull request merged**